learn/app-development/services/api-mock-server.md

```diff
@@ -38,7 +38,7 @@ Accessing API Mock Server to host a mocked API.
 
 - Go to the **Server**: https://apimock.wavemakeronline.com
 
-[![create mock api](/learn/assets/create-mock-api.png)](/learn/assets/create-mock-api.png)
+[![create mock api](/learn/assets/first-user-page.png)](/learn/assets/first-user-page.png)
```

Comment:
Could you please replace the image files with the same filename? This helps clean up our assets repo and replace images in other places if they're used elsewhere.